runtime/src/execute_cost_table.rs

```diff
@@ -15,10 +15,21 @@ const OCCURRENCES_WEIGHT: i64 = 100;
 
 const DEFAULT_CAPACITY: usize = 1024;
 
-#[derive(AbiExample, Debug)]
+// The coefficient represents the degree of weighting decrease in EMA,
+// a constant smoothing factor between 0 and 1. A higher alpha
+// discounts older observations faster.
+const COEFFICIENT: f64 = 0.4;
```

Comment:
Isn't this value too big?
This is effectively saying the latest observation has 40% weight in the estimate.
If I want my estimates to be roughly equivalent to average over a rolling window of length say 100, then each observation will have weight 1%. So I will need a coefficient here closer to 1%.

Reply:
> Isn't this value too big? This is effectively saying the latest observation has 40% weight in the estimate. If I want my estimates to be roughly equivalent to average over a rolling window of length say 100, then each observation will have weight 1%. So I will need a coefficient here closer to 1%.

Yeah, i am not sure how the alpha should be. If use `2/(N+1)` and take N=100, then alpha should be 1.98%. However, when I plug that into a spreadsheet with some sample data, comparing to straight average and Welford method, that diff is big. 0.4 makes it more close to current behavior (straight average).  Maybe can set it with 200 samples (alpha = 1%)